llpc/context/llpcCompiler.cpp

```diff
@@ -1259,9 +1265,32 @@ Result Compiler::buildPipelineInternal(Context *context, ArrayRef<const Pipeline
 
   context->setDiagnosticHandlerCallBack(nullptr);
 
+  if (result == Result::Success) {
+    const int llpcVersion[2] = {LLPC_INTERFACE_MAJOR_VERSION, LLPC_INTERFACE_MINOR_VERSION};
```

Comment:
Since this data is serialized/deserialized, give it an explicit width, e.g., `uint32_t`.

Reply:
done